examples/topics/graph/airports_graph.py

```diff
@@ -1,3 +1,15 @@
+'''
+This example creates an interactive map visualization of airport routes and locations using Bokeh.
```

Comment:
Start docstrings on the same line with a leading space, for consistency with existing examples (same comment in the other file in this PR as well)

```suggestion
''' This example creates an interactive map visualization of airport routes and locations using Bokeh.
```